services/notification/notifiers/mixins/message/sections.py

```diff
@@ -303,6 +317,12 @@ async def do_write_section(self, comparison, diff, changes, links, behind_by=Non
                     "Changes have been made to critical files, which contain lines commonly executed in production. [Learn more](https://docs.codecov.com/docs/impact-analysis)"
                 )
 
+        if comparison.all_tests_passed():
+            yield ("")
+            yield (
+                ":heavy_check_mark: Test ingestion set up successfully. No failed tests found. :relaxed:"
+            )
```

Comment:
Looks like this message will fire every time the `all_test_passed` check is true.  @giovanni-guidini  can you confirm?

Reply:
That is accurate.
Keep in mind that if the setup for test_results is complete but not all tests passed then we don't notify with this notifier.

Reply:
in that case... after first time setup ever time all test past, we'll show a message titled "Test ingestion set up successfully..."

Which is a little weird.. 

Maybe we alter that message to say "All tests successful" instead. wdyt @giovanni-guidini @Adal3n3

Reply:
@Adal3n3 @giovanni-guidini and @rohan-at-sentry aligned

Tweak the message to 

✅ All tests successful. No failed tests found :relaxed: